### PR TITLE
39851: New example file

### PR DESCRIPTION
**Pull request by labkey-steveh**
Create SampleSet domain and fields to an existing SampleSet. For issue [#39851](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=39851)

### CODE REVIEW

File: samples/sampleset_domain_example.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          'name': "Comments",
+          'rangeURI': "string"
+        }]
+    }
+}
+
+# domain.create returns the full Domain definition
+created_sampleset_domain = domain.create(server_context, sampleset_domain_definition)
+
+
```
Comment (labkey-klum) [inline samples/sampleset_domain_example.py]: remove this whitespace

File: samples/sampleset_domain_example.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+for field in inferred_fields:
+    sampleset_domain.add_field(field)
+
+domain.save(server_context, 'samples', 'BloodSamples', sampleset_domain)
+
+###################
+# Drop a domain
+###################
+
+#drop_response = domain.drop(server_context, 'samples', 'BloodSamples')
```
Comment (labkey-klum) [inline samples/sampleset_domain_example.py]: I would either uncomment these lines or delete them entirely

**Comment by labkey-nicka** [comment]
@labkey-steveh Should probably look to get this merged soon as we have other changes we're currently adding and can roll this into the version bump.

**Comment by labkey-steveh** [comment]
Thanks Karl, I made the changes you requested.

**Pull request merged**